app/helpers/hyrax/hyrax_helper_behavior.rb

```diff
@@ -211,6 +211,18 @@ def rights_statement_links(options)
       options[:value].map { |right| link_to service.label(right), right }.to_sentence.html_safe
     end
 
+    # A Blacklight facet field helper_method
+    # @param [String] value from blacklight helper_method invocation.
+    # @return [String] the decoded meaning of the boolean field
+    def suppressed_to_status(value)
+      case value
+      when "false"
+        "Published"
```

Comment:
I don't suppose you could be convinced to i18nize these strings?

Reply:
You are one tough reviewer.